refinery/ui/source/js/data-set-visualization/partials/visualization.html

```diff
@@ -74,7 +72,7 @@
                 <span ng-if="vis.owner.full_name.length > 2">
                   {{ vis.owner.full_name }}
                 </span>
-                <span ng-if="!vis.owner.full_name.length < 2">
+                <span ng-if="!(vis.owner.full_name.length < 2)">
```

Comment:
I'm still confused here... Is the intention to have this and the condition above be mutually exclusive?

Reply:
@mccalluc Gotcha. Thanks